ci/Dockerfile

```diff
@@ -1,20 +1,10 @@
-FROM bepsays/ci-goreleaser:1.12-2
+FROM circleci/golang:1.12-buster 
```

Comment:
Could you use 1.13-buster?

Reply:
I think upgrading go version is another PR and needs more testing

Reply:
I've already worked on supporting Go 1.13. I think it should work with Go 1.13. But another PR is OK.